tst/EnergyPlus/unit/OutputProcessor.unit.cc

```diff
@@ -3355,6 +3355,72 @@ namespace EnergyPlus {
 			} ) );
 
 		}
+		TEST_F (SQLiteFixture, OutputProcessor_GenOutputVariablesAuditReport)
```

Comment:
This unit test doesn't seem to test the code changes in this PR. You don't have any `Meter:Custom` in the IDF snippet. Is this unit test for a different pull request?